aten/src/TH/vector/avx_mathfun.h

```diff
@@ -68,7 +68,7 @@ _PS256_CONST_TYPE(min_norm_pos, int, 0x00800000);
 _PS256_CONST_TYPE(mant_mask, int, 0x7f800000);
 _PS256_CONST_TYPE(inv_mant_mask, int, ~0x7f800000);
 
-_PS256_CONST_TYPE(sign_mask, int, 0x80000000);
+_PS256_CONST_TYPE(sign_mask, int, (int)0x80000000);
```

Comment:
Why is this cast necessary? It's not used for any other literals

Reply:
Without the cast there is a `error: constant expression evaluates to 2147483648 which cannot be narrowed to type 'int' [-Wc++11-narrowing]` error. I think `0x80000000` is actually over the range for int, and originally it was intended to be evaluated to `-2147483648`, not `2147483648` (as it would be if it's an unsigned int by default).

Reply:
I'm not 100% sure that adding the cast actually fixes it. Overflow in signed integers is UB, so this value can be really anything.

Reply:
Hmm what should be the right fix here? @goldsborough Do you mind taking a look?